app/javascript/packs/application.js

```diff
@@ -23,6 +24,13 @@ document.addEventListener('DOMContentLoaded', () => {
   Vue.component(VueQrcode.name, VueQrcode);
   Vue.use(VModal);
   Vue.use(VueClipboard);
+  Vue.use(ActionCableVue, {
+    debug: true,
+    debugLevel: 'error',
+    connectionUrl: process.env.RAILS_ENV === 'production' ?
+    process.env.APPLICATION_HOST.replace('https', 'wss') : 'ws://localhost:3000/cable',
```

Comment:
Expected indentation of 6 spaces but found 4                        indent<br>'process' is not defined                                            no-undef